src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -334,6 +334,12 @@ private void PendSubmissions(IEnumerable<PendingSubmission> inputs)
 
             public void AddInput(string command)
             {
+                // If the language buffer is readonly then input can not be added. Return immediately.
+                if (_window._currentLanguageBuffer != null && _window._currentLanguageBuffer.IsReadOnly(0))
```

Comment:
When does this occur?  Would it be appropriate to simply add a new, writeable language buffer, as when it's null?